azure-pipelines.yml

```diff
@@ -406,24 +406,7 @@ jobs:
         - sw_build
   - bash: |
       set -e
-      set -x
-      . util/build_consts.sh
-      SHA=$(git rev-parse HEAD)
-      BITCACHE_DIR=~/.cache/opentitan-bitstreams/cache/${SHA}
-      mkdir -p $BITCACHE_DIR
-      BITCACHE_FILE=$BITCACHE_DIR/lowrisc_systems_chip_earlgrey_cw310_0.1.bit.orig
-      cp $BIN_DIR/hw/top_earlgrey/lowrisc_systems_chip_earlgrey_cw310_0.1.bit ${BITCACHE_FILE}
-      echo "" >> ${BITCACHE_DIR}/lowrisc_systems_chip_earlgrey_cw310_0.1.bit.splice
-      echo -n ${SHA} > ${BITCACHE_DIR}/../../latest.txt
-      export BITSTREAM="--offline --list ${SHA}"
-      ci/bazelisk.sh test \
-          --define DISABLE_VERILATOR_BUILD=true \
-          --test_tag_filters=cw310,-broken \
-          --test_output=all \
-          --build_tests_only \
-          --define cw310=lowrisc \
-          $(./bazelisk.sh query 'rdeps(//...,@bitstreams//:bitstream_test_rom)') \
-          # All the tests that depend on the test rom
+      ci/scripts/run-fpga-cw310-tests.sh
```

Comment:
Nit can you file a follow-on issue: moving this to a script isn't enough to make it portable but implies it is. This script wont work without the bitstream and isn't testing for it's presence.

Reply:
Moving it to a script is likely to become permanent because it's useful.

Reply:
Yeah I think I will need to some more work to figure out where it's searching for the bitstream.

Reply:
This is now tracked in #13070